src/core/execute.h

```diff
@@ -135,6 +135,7 @@ typedef enum ExecDirectoryType {
 typedef struct ExecDirectoryItem {
         char *path;
         char **symlinks;
+        bool only_create;
 } ExecDirectoryItem;
```

Comment:
so far we tried to keep runtime data and configured settings in seperate structs (though there are exceptions, i.e. ExecCommand is config data but embedds ExecStatus which is runtime data). Which makes me a bit concerned about embedding this bool here given it's only used internally, for a brief moment even...

Reply:
I do not think this is runtime data. This is a kind of preprocessed data from unit files and/or dbus operation for code optimization, and purely calculated from them. This just caches the existence of one of parent directories in the list.
Without the flag, we need to search the list twice in `setup_exec_directory()` and `compile_bind_mounts()`.